examples/cmake-examples/QmlBarcodeGenerator/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 3.16.3)
```

Comment:
Splitting examples into two separate directories for CMake and qmake is not really a good idea. There is a high code redundancy due to the fact that both of the cmake and qmake directories contains most of the same source code. This leads to need of modifying two codebases every time examples need to be updated.

You can place examples in a single directory and place both `.pro` and `CMakeLists.txt` in the same directory so they can use the same codebase. To avoid confusion add a mention in README.md that for both of the build systems have their examples located in the same directory and all you need to do is to just open proper file for different build system to be used